wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -29,6 +29,9 @@ <h1 class="w-header__title">
                         <div class="w-header__glyph avatar"><img src="{{ avatar }}" alt="" /></div>
                     {% endif %}
                     {{ title }}{% if subtitle %} <span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
+                    {% if header_description %}
+                        <div class="w-header__description">{{ header_description }}</div>
+                    {% endif %}
                 </h1>
```

Comment:
I think the description should be outside the h1 tag. 

Technically the subtitle should also - but that is a legacy decision we shouldn't change now. 

Remember that the h1 tag is not just a visual thing but a critical part of the page's context given to screen readers and other assistive technology. 

Adding a description of the page is useful but it's not really part of the h1 title.